.github/dependabot.yml

```diff
@@ -6,692 +6,8 @@ version: 2
 
 updates:
 # go.mod
-- package-ecosystem: "gomod"
```

Comment:
In this case, there should be no changes to this file. Something broke when running `make generate` locally. Try to revert the changes to this file and run `make generate` again.

Reply:
Done